guides/common/modules/proc_configuring-group-mapping-for-keycloak-authentication.adoc

```diff
@@ -1,20 +1,20 @@
 [id="configuring-group-mapping-for-keycloak-authentication_{context}"]
-= Configuring group mapping for {keycloak} authentication
+= Optional: Configuring external group mapping for {keycloak} authentication
 
-Optionally, to implement the Role Based Access Control (RBAC), create a group in {Project}, assign a role to this group, and then map an Active Directory group to the {Project} group.
-As a result, anyone in the given group in {keycloak} are logged in under the corresponding {Project} group.
-This example configures users of the {Project}-admin user group in the Active Directory to authenticate as users with administrator privileges on {Project}.
+To implement the Role Based Access Control (RBAC), create a group in {Project}, assign a role to this group, and then map an Active Directory group to the {Project} group.
```

Comment:
```suggestion
To implement the Role Based Access Control (RBAC), create a group in {Project}, assign a role to this group, and then map an {keycloak} group to the {Project} group.
```

As it's about leveraging groups from the external system, Keycloak groups should be more correct here, right? Yes of course you might have a federation from Keycloak to AD, but I see some possible confusion here, when you have configured both AD and Keycloak, and the user might expect being able to link groups from AD even if they are not brought with the group mapping from Keycloak. Maybe I'm also just overthinking this 🙂